02_operation_of_running_systems/110_manage_startup_services/111_solution.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+sudo systemctl restart crond
```

Comment:
I think you should add a status command to double check the proper restart of crond

Reply:
Done